.github/workflows/bencher.yml

```diff
@@ -186,6 +186,12 @@ jobs:
         run: cargo gen-spec
       - name: Check OpenAPI Spec for changes
         run: git diff --exit-code
+      - name: Generate Typescript Types
+        run: |
+          cargo install typeshare-cli --version 1.13.2 --locked
```

Comment:
Let's break this out into a separate step. It makes it a lot easier to glance and see what is going on in GitHub Actions.